app/config/import.js

```diff
@@ -0,0 +1,47 @@
+const {writeFileSync, readFileSync} = require('fs');
+const {sync: mkdirpSync} = require('mkdirp');
+const _paths = require('./paths');
+const _init = require('./init');
+const _keys = require('./keymaps');
+
+const _write = function (path, data) {
+  // This method will take text formatted as Unix line endings and transform it
+  // to text formatted with DOS line endings. We do this because the default
+  // text editor on Windows (notepad) doesn't Deal with LF files. Still. In 2017.
+  const crlfify = function (str) {
+    return str.split('\n').map(x => x.indexOf('\r') < 0 ? x : `${x}\r`).join('\n');
+  };
+  const format = process.platform === 'win32' ? crlfify(data) : data;
+  writeFileSync(path, format);
+};
+
+const _importConfig = function () {
+  try {
+    return readFileSync(_paths.preferencesPath, 'utf8');
+  } catch (err) {
+    try {
+      const cfg = readFileSync(_paths.previousConfigPath);
+      _write(_paths.preferencesPath, cfg);
+      return cfg;
+    } catch (err) {
+      const cfg = readFileSync(_paths.dotConfigPath);
+      _write(_paths.preferencesPath, cfg);
+      return cfg;
+    }
+  }
+};
+
+const _importPlugins = function () {
+  mkdirpSync(_paths.pluginsPath);
+  mkdirpSync(_paths.localPluginsPath);
+};
```

Comment:
This isn't a great name for this function, you aren't importing plugins so much as making the plugins directories.